pkg/smartcontract/manifest/abi.go

```diff
@@ -58,7 +59,7 @@ func (a *ABI) IsValid() error {
 	for i := range a.Methods {
 		err := a.Methods[i].IsValid()
 		if err != nil {
-			return err
+			return fmt.Errorf("ABI method #%d ('%s'/%d): %w", i, a.Methods[i].Name, len(a.Methods[i].Parameters), err)
```

Comment:
Do we need `i` here? Usually people have no idea what's the position of some method in this JSON.

Reply:
And it's be nice to use `%q` instead of `'%s'`, Name can have any kind of weird characters inside.